0038-http-auth-profiles/http-auth-profiles.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Interledger HTTP Authentication Profiles
+draft: 1
+---
+
+# Interledger HTTP Authentication Profiles
+A minimal set of extensible, token-based profiles for securing Interledger relationships using HTTP.
+
+## Motivation
+Many Interledger protocols and software work well using HTTP. For example, [ILP-over-HTTP](https://github.com/interledger/rfcs/blob/master/0035-ilp-over-http/0035-ilp-over-http.md) utilizes HTTP at the [Link Layer](https://interledger.org/rfcs/0001-interledger-architecture/). Additionally, Interledger software often uses HTTP for administrative functionality and other purposes.
+
+This document defines common profiles that can be used to ensure implementations of Interledger software work securely in an interoperable fashion, while simultaneously allowing Interledger software operators to more clearly understand the trade-offs involved with various security choices.
+
+The profiles defined in this RFC should:
+
+* ...be as simple as possible while also being highly secure. 
```

Comment:
The term "highly secure" makes me cringe a bit. Everything that follows is about security but each make different tradeoffs and assume different threat models

Reply:
> The term "highly secure" makes me cringe a bit

I was thinking about wording tweaks, but am now thinking we can just strike this sentence.

Reply:
Removed.